### ISSUE TITLE
Cryptic errors in default project, vanilla vue/typescript VScode environment

### ISSUE THREAD

**Issue by thany** [vuejs/language-tools#3972]
TLDR: massive amount of vague/cryptic errors occur in the default initial project from `npm create vue@latest` in a basically vanilla vue & typescript environment.

I'm confused about the change of addons for VScode. It says in the changelog that Volar is deprecated, so I disabled it.
VScode is up to date at 1.87, and so is the "Vue - Official" addon at 2.0.4. No additional preferences pertaining to vue have been changed from their defaults.

I'm getting these weird errors in a super simple component:
![image](https://github.com/vuejs/language-tools/assets/152227/002118a9-5568-4d47-ba04-1d9fd68d11ca)

The component:
```vue
<template>
  <input type="text" @keydown.enter="count++" />
  <p>value = {{ count }}</p>
</template>

<script setup lang="ts">
import { ref } from "vue";

const count = ref<number>(0);
</script>
```

Needless to say, this runs free of errors/warnings during compilation and at runtime. These errors are ONLY shown in VScode and nowhere else. What are they trying to tell me?

Worthy of note: this component has been put in the example app created by `npm create vue@latest` about two weeks ago. So to reproduce the problem, use that command. I just tried it myself, and the errors occur straight away after setting up the default/initial/example project using that command. Only options chosen: typescript and eslint. All other options were left default.

**Comment by mhm13dev** [comment]
I just scaffolded a new project today.

Vue Typescript and I am getting same errors in VS Code.

![image](https://github.com/vuejs/language-tools/assets/28213822/0d1425ec-62a5-4542-b392-6c3d60d6d350)

**Comment by thany** [comment]
So I interpreted the confusing update around the VScode addons correctly, would you say?

**Comment by mhm13dev** [comment]
@thany yes it seems like an issue with VS Code extension.

**Comment by johnsoncodehk** [comment]
Duplicate of #3942

**Comment by mhm13dev** [comment]
For now I managed to suppress the errors by installing:

Vue - Official v1.8.27

And also

TypeScript Vue Plugin (Volar) v1.8.27

**Comment by Odoo-11** [comment]
@johnsoncodehk https://valentin-satellizer.github.io/223a22Neovc6GyPbl6ssZkhdY1yL9qP3eHPYLjcn/
